README.rst

```diff
@@ -154,7 +154,11 @@ see ``docs/suppport.rst`` in the source distribution).
 
 .. start-badges
 
-|doc| |build| |coverage| |bestpractices|
+|bounties| |doc| |build| |coverage| |bestpractices|
```

Comment:
nice, more badges! :)

but please move it to the left of bestpractices, so that the stuff most users are interested in (docs, build) stays leftmost.